package.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Can we move this somewhere into `.github/workflows` or `.github/actions`? I'm not sure what the right pattern for this is, but given that we have more files now it would be better to have everything contained there. Same for `changelog.js`.